libs/components/indicators/src/lib/modules/label/label.component.spec.ts

```diff
@@ -32,4 +32,17 @@ describe('Label component', () => {
     await fixture.whenStable();
     await expectAsync(fixture.nativeElement).toBeAccessible();
   });
+
+  it('should default to "info" when no alert type is specified', async () => {
+    const fixture = TestBed.createComponent(LabelTestComponent);
+    const cmp = fixture.componentInstance as LabelTestComponent;
+    const el = fixture.nativeElement as HTMLElement;
+
+    cmp.labelType = undefined;
+    fixture.detectChanges();
+
+    const alertEl = el.querySelector('.sky-label');
+    expect(alertEl?.classList.contains('sky-label-info')).toBe(true);
+    expect(fixture.nativeElement).toBeAccessible();
+  });
```

Comment:
This is needed for 100% code coverage of the `SkyLabelComponent`.

Reply:
Could you back this one out? I am already addressing it in a separate PR.